packages/notifications/src/styled/StyledBase.ts

```diff
@@ -55,17 +91,22 @@ const padding = (props: ThemeProps<DefaultTheme>) => {
   return `${paddingVertical} ${paddingHorizontal}`;
 };
 
-export const StyledBase = styled.div`
+export const StyledBase = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
```

Comment:
I remember we had some back and forth originally, and we decided to leave out the details of the base component because we expected all other icon components (like this one) to define the `COMPONENT_ID` AND `retrieveComponentStyles` etc.